src/gtest/test_checkblock.cpp

```diff
@@ -268,11 +260,12 @@ TEST_F(ContextualCheckBlockTest, BlockSproutRulesRejectSaplingTx) {
 
     SCOPED_TRACE("BlockSproutRulesRejectSaplingTx");
```

Comment:
The previous `SCOPED_TRACE` won't have fallen out of scope. Will this overwrite it? (Similarly below.)